index.html

```diff
@@ -0,0 +1,194 @@
+            <form id="form" action="https://www.freecodecamp.com/email-submit">
+              <div class="form-group my-2 d-grid">
+                <input name="email" id="email" type="email" placeholder="Enter your email" required>
+                <button id="submit" type="submit" value="Submit your Email" class="btn mx-auto btn-success mt-4">Submit your Mail</button>
+              </div>
+          </div>
+        </div>
+      </div>
+    </div>
+   </section>
+   <section id="features">
+    <div class="container">
+      <div class="row">
+        <div class="group1 d-flex justify-content-around">
+          <i class="fa fa-truck fa-2x"></i>
+          <div class="group1-content">
+            <h2>Premium Materials</h2>
+            <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+          </div>
+        </div>
+      
+        <div class="group1 d-flex justify-content-around">
+          <i class="fa fa-truck fa-2x"></i>
+          <div class="group1-content">
+            <h2>Premium Materials</h2>
+            <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+          </div>
+
+        </div>
+      
+        <div class="group1 d-flex justify-content-around">
+          <i class="fa fa-truck fa-2x"></i>
+          <div class="group1-content">
+            <h2>Premium Materials</h2>
+            <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+          </div>
+
+        </div>
+
+        <div class="row">
+          <div class="col-md-12 d-flex justify-content-center">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+          </div>
+        </div>
+      
+      </div>
+    </div>
+    </section>
+    <section class="my-5" id="product-card">
+          <div class="row">
+            <div class="col-md-4 d-flex justify-content-around">
+              <div class="card text-center">
+                <div class="card-header">
+                  <img class="img-responsive" height="200" src="https://cdn.pixabay.com/photo/2017/08/17/08/20/online-shopping-2650383_960_720.jpg" alt="product-image">
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title">$600</h3>
+                    <p>
+                      <ul>
+                        <li>Lorem ipsum dolor sit amet consectetur</li>
+                        <li>Lorem ipsum dolor sit amet consectetur</li>
+                        <li>Lorem ipsum dolor sit amet consectetur</li>
+                        <li>Lorem ipsum dolor sit amet consectetur</li>
+                      </ul>
+                    </p>
```

Comment:
Please change the tag you used here. I would recommend you using a div tag instead of a p tag ,because it will be more general and make your code semantically correct.